ion-c-sys/src/reader.rs

```diff
@@ -23,11 +23,10 @@ use crate::string::IonCStringRef;
 /// # use std::convert::*;
 /// # use std::ptr;
 /// # fn main() -> IonCResult<()> {
-/// let mut buf = b"hello".to_vec();
-/// let mut reader = IonCReaderHandle::try_from(buf.as_mut_slice())?;
+/// let mut reader = IonCReaderHandle::try_from(b"\xE0\x01\x00\xEA\x85hello".as_ref())?;
```

Comment:
Forgive the stupid question: this byte sequence shows up all over the place--is it just the [binary version marker](http://amzn.github.io/ion-docs/docs/binary.html#value-streams)?

Reply:
Yes, this sequence is the BVM: `E0 01 00 EA`